aries-site/src/examples/components/namevaluelist/Examples/NameValueListPairPropsExample.js

```diff
@@ -9,7 +9,10 @@ export const NameValueListPairPropsExample = () => (
         let date;
         if (name === 'Created on') {
           const event = new Date(value);
-          date = event.toLocaleDateString();
+          date = event.toLocaleString('en-US', {
+            dateStyle: 'short',
```

Comment:
I've heard comments from Edgar before that Greenlake tries to "zero pad" dates (I'm not sure about times) to make scanning multiple lines easier. So, for example 01/23/21, 7:15 AM. Should we do that? @halocline

Reply:
`10/5/21, 8:48 AM`
This is what the preview is currently showing so is that not what this is  01/23/21, 7:15 AM?

Reply:
I think we can leave it to separate work handled here: https://github.com/grommet/hpe-design-system/issues/2039

Reply:
got it I was a little confused